src/components/Confirm/CaptureViewer/style.css

```diff
@@ -17,7 +17,7 @@
 .videoWrapper {
   display: flex;
   position: relative;
-  height: 270px;
+  height: 53.35%;
```

Comment:
this seems to be override with `height: 306px;` when I try it on chrome

Reply:
is that expected?

Reply:
That's expected because the `.videoWrapper` has a height in px. For consistency, I will add the height in percentage also for the video wrapper, even if video is not affected by this issue because the old browsers I've tested do not support video recording/getUserMedia.

Reply:
so on which browsers will this % be actually used?

Reply:
This feature is not browser specific. We will use % for `.imageWrapper`, `.pdfWrapper` and `.videoWrapper`. The SDK is way more responsive now.